sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -311,6 +311,27 @@ extern __DPCPP_SYCL_EXTERNAL void __spirv_CooperativeMatrixPrefetchINTEL(
     T *Ptr, uint32_t NumRows, uint32_t NumCols, unsigned int CacheLevel,
     __spv::MatrixLayout Layout, size_t Stride);
 
+template <typename T, typename Tp, std::size_t R, std::size_t C,
+          __spv::MatrixUse U,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern __DPCPP_SYCL_EXTERNAL
+    __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *
```

Comment:
Hello Dounia! I found this line needs to be updated to `__spv::__spirv_CooperativeMatrixKHR<Tp, S, R, C, U> *` because a commit merged two weeks ago switched operations to use SPV_KHR_cooperative_matrix (https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2804)